### PR TITLE
added chattypeEntity

### PR DESCRIPTION
**Pull request by nazarmavko**

### CODE REVIEW

File: src/main/java/com/softserve/actent/model/entity/ChatType.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package com.softserve.actent.model.entity;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+import javax.persistence.*;
```
Comment (RBarnabass) [inline src/main/java/com/softserve/actent/model/entity/ChatType.java]: You should set exactly what you import

File: src/main/java/com/softserve/actent/model/entity/ChatType.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import javax.validation.constraints.NotBlank;
+
+@Entity
```
Comment (RBarnabass) [inline src/main/java/com/softserve/actent/model/entity/ChatType.java]: An entity can have a table name and you can remove table annotation

File: src/main/java/com/softserve/actent/model/entity/ChatType.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+@Table(name="chat_types")
+@NoArgsConstructor
+public class ChatType {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NonNull
+    @NotBlank(message="type can`t be empty.")
+    @Column(name="type",unique = true)
```
Comment (octap16) [inline src/main/java/com/softserve/actent/model/entity/ChatType.java]: The name of the field is the same with Column name

**Review by Senchakmen — DISMISSED** [review]

File: src/main/java/com/softserve/actent/model/entity/ChatType.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Column;
+import javax.validation.constraints.NotBlank;
+
+@Entity
```
Comment (onlyVone-zz) [inline src/main/java/com/softserve/actent/model/entity/ChatType.java]: Entity and table annotations may be last

File: src/main/java/com/softserve/actent/model/entity/ChatType.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+@Table(name="chat_types")
+@NoArgsConstructor
+public class ChatType {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NonNull
+    @NotBlank(message="type can`t be empty.")
+    @Column(unique = true,nullable = false)
```
Comment (RBarnabass) [inline src/main/java/com/softserve/actent/model/entity/ChatType.java]: add spaces between = and other things

File: src/main/java/com/softserve/actent/model/entity/ChatType.java [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+@Entity
+@Table(name="chat_types")
+public class ChatType {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NonNull
+    @NotBlank(message="type can`t be empty.")
+    @Column(unique = true,nullable = false)
```
Comment (md1guy) [inline src/main/java/com/softserve/actent/model/entity/ChatType.java]: spaces around commas and equals sign

**Review by md1guy — DISMISSED** [review]

**Review by RBarnabass — APPROVED** [review]

**Review by thebadpanda — APPROVED** [review]

**Review by md1guy — APPROVED** [review]

**Pull request merged**